sdk/communication/communication-phone-numbers/src/models.ts

```diff
@@ -76,12 +77,18 @@ export interface SearchOperatorInformationOptions extends OperationOptions {
 /**
  * Additional options that can be passed to list SIP routes.
  */
-export interface ListSipRoutesOptions extends OperationOptions {}
+export interface ListSipRoutesOptions extends OperationOptions {
```

Comment:
Not needed, we can call Get for routes always without the ExpandEnum parameter, because routes don't carry any info on trunk health